packages/fast-components-styles-msft/src/select/index.ts

```diff
@@ -13,59 +17,78 @@ import {
+): ComponentStyleSheet<SelectClassNameContract, DesignSystem> => {
+    const designSystem: DesignSystem = withDesignSystemDefaults(config);
+    const direction: Direction = designSystem.direction;
+
+    return {
+        select: {
+            minWidth: "276px",
+            maxWidth: "374px",
+        },
+
+        select_button: {
+            height: height(),
+            width: "100%",
+            ...inputFieldStyles(),
+        },
 
-const styles: ComponentStyles<SelectClassNameContract, DesignSystem> = {
-    select: {
-        minWidth: "276px",
-        maxWidth: "374px",
-    },
+        select_buttonContentRegion: {
+            display: "grid",
+            gridTemplateColumns: "1fr auto",
+            alignItems: "center",
+            justifyItems: "start",
+        },
 
-    select_toggleGlyph: {
-        fill: neutralForegroundRest,
-    },
+        select_buttonDisplayText: {
+            ...ellipsis(),
+            textAlign: applyLocalizedProperty("left", "right", direction),
+            width: "100%",
+        },
 
-    select__disabled: {
-        opacity: ".3",
-    },
+        select_toggleGlyph: {
+            [applyLocalizedProperty(
+                "margin-left",
+                "margin-right",
+                direction
+            )]: horizontalSpacing(),
+            fill: neutralForegroundRest,
+            width: glyphSize,
+            height: glyphSize,
+            gridColumnStart: "2",
+        },
 
-    select_menu: {
-        background: neutralFillStealthRest,
-        ...applyFloatingCornerRadius(),
-        ...applyElevation(ElevationMultiplier.e11),
-        zIndex: "1",
-        position: "absolute",
-        width: "100%",
-        margin: "0",
-        padding: format("{0} 0", toPx<DesignSystem>(designUnit)),
-        maxWidth: "374px",
-        minWidth: "276px",
-        maxHeight: "328px",
-        overflow: "auto",
-    },
+        select__disabled: {},
+
+        select_menu: {
+            background: neutralFillStealthRest,
+            ...applyFloatingCornerRadius(),
+            ...applyElevation(ElevationMultiplier.e11),
+            zIndex: "1",
+            position: "absolute",
+            width: "100%",
+            margin: "0",
+            padding: format("{0} 0", toPx<DesignSystem>(designUnit)),
+            maxWidth: "374px",
+            minWidth: "276px",
+            maxHeight: "328px",
+            overflow: "auto",
+        },
 
-    select__multiSelectable: {
-        "& $select_menu": {
-            position: "static",
-            boxShadow: "none",
-            border: "1px solid",
-            borderColor: neutralOutlineRest,
+        select__multiSelectable: {
+            "& $select_menu": {
+                position: "static",
+                boxShadow: "none",
+                border: "1px solid",
+                borderColor: neutralOutlineRest,
+            },
         },
-    },
-    select_menu__open: {},
+        select_menu__open: {},
```

Comment:
What was the reason for this class again?

Reply:
It is intended for implementations where the trigger might look differently when the menu is open, ours doesn't, but doing things like rotating the icon when the menu is open are common.  So this implementation doesn't require this in the stylesheet and I'll remove it.